### DISCUSSION TITLE
not sure what happened but it keeps repeat these code

### DISCUSSION THREAD

**Discussion by AutumnRM** [LSPosed/MagiskOnWSALocal#294] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/93046016/206950035-a0628cb9-620e-46fd-985c-451cdd20583e.png)
at first everything went well,i selected wsa Magisk gapps and things,entered admin password
and i realize something was wrong after looking at these code repeating for like ten minutes
ping GitHub.com went well so unlikely to be network problems
help😭

**Comment by Howard20181** [comment, marked as answer]
Fail to download Magisk from Github, you can try to download yourself then place to correct path.
